jsconfig.json

```diff
@@ -0,0 +1,15 @@
+{
+  "compilerOptions": {
+    "target": "ES2020",
+    "module": "ESNext",
+    "allowJs": true,
+    "checkJs": true,
+    "moduleResolution": "node",
+    "baseUrl": "./",
+    "paths": {
+      "@/*": ["src/*"]
+    }
```

Comment:
`@/`-alias imports are not used within the Blits code base

Reply:
ah right - this my editor being smart. Will remove! thanks